woocommerce-gateway-ebanx/woocommerce-gateway-ebanx.php

```diff
@@ -251,25 +251,43 @@ public function my_account_template()
 		{
 			if (isset($_POST['credit-card-delete']) && is_account_page()) {
 				// Find credit cards saved and delete the selected
-				$cards = get_user_meta(get_current_user_id(), '_ebanx_credit_card_token', true);
+				$credit_cards = get_user_meta(get_current_user_id(), '_ebanx_credit_card_token', true);
 
-				foreach ($cards as $k => $cd) {
+				foreach ($credit_cards as $k => $cd) {
 					if ($cd && in_array($cd->masked_number, $_POST['credit-card-delete'])) {
-						unset($cards[$k]);
+						unset($credit_cards[$k]);
 					}
 				}
 
-				update_user_meta(get_current_user_id(), '_ebanx_credit_card_token', $cards);
+				update_user_meta(get_current_user_id(), '_ebanx_credit_card_token', $credit_cards);
 			}
+			
+			if (isset($_POST['debit-card-delete']) && is_account_page()) {
+				// Find debit cards saved and delete the selected
+				$debit_cards = get_user_meta(get_current_user_id(), '_ebanx_debit_card_token', true);
+
+				foreach ($debit_cards as $k => $cd) {
+					if ($cd && in_array($cd->masked_number, $_POST['debit-card-delete'])) {
```

Comment:
We're not using superglobals anymore, please replace `$_POST[<value>]` with `WC_EBANX_Request::read(<value>)`